rcl/include/rcl/context.h

```diff
@@ -132,7 +139,7 @@ typedef struct rcl_context_t
    * See this paper for an effort to make this possible in the future:
    *   http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0943r1.html
    */
-  uint8_t instance_id_storage[RCL_CONTEXT_ATOMIC_INSTANCE_ID_STORAGE_SIZE];
+  ALIGNAS_(8) uint8_t instance_id_storage[RCL_CONTEXT_ATOMIC_INSTANCE_ID_STORAGE_SIZE];
```

Comment:
Does `RCL_CONTEXT_ATOMIC_INSTANCE_ID_STORAGE_SIZE` instead of `8` work?

Reply:
Unfortunately not, this will result in `__declspec(align(sizeof(uint_least64_t)))` and MSVC doesn't like that, giving `syntax error: 'sizeof'`: https://godbolt.org/z/vskBhM